core/Names.cc

```diff
@@ -243,6 +243,14 @@ string NameRef::show(const GlobalState &gs) const {
     return data(gs)->show(gs);
 }
 
+NameRef NameRef::originalName(const GlobalState &gs) const {
```

Comment:
Please don't introduce globally accessible methods without a clear semantic usecase for them outside of a signle module.